### PR TITLE
Kne link flap

### PR DESCRIPTION
**Pull request by abhijit-dhar**
feature/gribi/otg_tests/ipv4_entry_test/ipv4_entry_test.go has been changed. Due to addition of the feature kne link flap this test needs to be changed. Main chaange is there should not be any set config/push config call after link flap has been triggered.

### CODE REVIEW

**Review by jasdeep-hundal — CHANGES_REQUESTED** [review]

File: feature/gribi/otg_tests/weighted_balancing_test/port_flap_rebalanced_test.go
[the remark this answers is not in the document]
Comment (jasdeep-hundal) [reply feature/gribi/otg_tests/weighted_balancing_test/port_flap_rebalanced_test.go:1]: @abhijit-dhar : This still needs to be addressed.

**Review by jasdeep-hundal — CHANGES_REQUESTED** [review]

File: feature/gribi/otg_tests/weighted_balancing_test/port_flap_rebalanced_test.go
[the remark this answers is not in the document]
Comment (abhijit-dhar) [reply feature/gribi/otg_tests/weighted_balancing_test/port_flap_rebalanced_test.go:1]: Hi Jasdeep, there are several file where this deviation exists. Can you please merge this PR with the deviation  ATEPortLinkStateOperationsUnsupported there? Then later we can log a separate PR with a request to remove all such deviations after merging this PR.

[the remark this answers is not in the document]
Comment (jasdeep-hundal) [reply feature/gribi/otg_tests/weighted_balancing_test/port_flap_rebalanced_test.go:1]: @abhijit-dhar : Happy to defer these changes. Please sign the CLA and I can get this merged as-is.

**Review by jasdeep-hundal — CHANGES_REQUESTED** [review]
@abhijit-dhar : Actually needs formatting fixes as well (should just need to run `gofmt` on the files).

**Comment by jasdeep-hundal** [comment]
Closed in favor of https://github.com/openconfig/featureprofiles/pull/2573

**Pull request closed without merging**